mailpoet/lib/Newsletter/BlockPostQuery.php

```diff
@@ -168,10 +168,13 @@ public function getQueryParams(): array {
     $parameters['suppress_filters'] = false;
 
     if ($this->newerThanTimestamp instanceof DateTimeInterface) {
+      //Ensure UTC timezone
+      $after = new \DateTime('now', new \DateTimeZone('UTC'));
+      $after->setTimestamp($this->newerThanTimestamp->getTimestamp());
```

Comment:
[WordPress sets PHP timezone to `UTC`](https://github.com/WordPress/WordPress/blob/7b9673722e9f9198ae02a17465e2c562c08d5815/wp-settings.php#L71) so the `$this->newerThanTimestamp` should be in UTC, but it makes no harm to add some extra safety here.